cai_causal_graph/time_series_causal_graph.py

```diff
@@ -286,10 +338,10 @@ def extend_graph(
         # check steps are valid (positive integers) if not None
         if backward_steps is not None:
             assert backward_steps == int(backward_steps), f'backward_steps must be an integer. Got {backward_steps}.'
-            assert backward_steps > 0, f'backward_steps must be a positive integer. Got {backward_steps}.'
+            assert backward_steps >= 0, f'backward_steps must be greater than zero. Got {backward_steps}.'
```

Comment:
The message says greater than zero, but is checks it is greater or equal. 

Also, I still think it needs to be an integer no? What if someone passes a float?

Reply:
If someone passes a float it is fine. See line above: `assert backward_steps == int(backward_steps), f'backward_steps must be an integer. Got {backward_steps}.'`

But the message should be "must be a non-negative integer."

Reply:
Sorry, missed that line!